terraformer_policy.tf

```diff
@@ -1,72 +1,35 @@
 # Create the IAM policy for the Terraformer EC2 server instances that
 # allows full access to create/destroy new resources in this account
 # and create/modify/destroy existing resources that _are not_ tagged
-# as being created by the team that deploys this root module.
+# as being created by the team that deploys this root module (there are
+# some exceptions to this rule, see below for details).
 #
 # Also allow sufficient permissions to launch instances in the
 # operations subnet and use the existing security groups.
 
 data "aws_iam_policy_document" "terraformer_policy_doc" {
   provider = aws.provisionassessment
 
-  # Allow full access to new resources and existing resources that
-  # _are not_ tagged as being created by the team that deploys this
-  # root module, with the exception of IAM.
-  #
-  # We will attach the arn:aws:iam::aws:policy/ReadOnlyAccess policy
-  # to the same role to which the policy document will be attached, which
-  # will give it read-only access to all IAM resources.  We require
-  # IAM access to be as read-only as possible in order to stop the
-  # Terraformer instance from defeating the Terraformer policy by
-  # creating new users, policies, roles, etc.
+  # Deny modification of any resources tagged by the team that deploys this
```

Comment:
I'm not a fan of this phrasing because there's no explicit `Deny` going on here. I would rather frame it terms of what is allowed since this is an allow statement.

Reply:
Please take a look at https://github.com/cisagov/cool-assessment-terraform/pull/215/commits/b7a9b68396c9557f73bcc6601cff39e4bc64f282 and let me know what you think.